gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/grouping/Grouping.java

```diff
@@ -379,17 +466,23 @@ public GroupingBuilder setStrategy(GroupingStrategy strategy) {
     }
 
     /**
-     * Adds a property key to the vertex grouping keys.
+     * Adds a property key to the vertex grouping keys, or adds a new label group to the vertex
+     * label groups.
      *
-     * @param key property key
+     * @param key either property key or label if propertyKeys are used
+     * @param propertyKeys property keys used for different labels
      * @return this builder
      */
-    public GroupingBuilder addVertexGroupingKey(String key) {
+    public GroupingBuilder addVertexGroupingKey(String key, String... propertyKeys) {
```

Comment:
No, this calls create two groups for vertices with the label "Person", if you want to use multiple keys you can append these like this `builder.addVertexGroupingKey("Person", "age", "name);`